rero_ils/modules/items/api/api.py

```diff
@@ -135,6 +139,27 @@ def reasons_not_to_delete(self):
             )
         return cannot_delete
 
+    def in_collection(self, **kwargs):
+        """Get published collection pids for current item."""
+        from ...collections.api import CollectionsSearch
+        output = []
+        result = list(
+            CollectionsSearch()
+            .filter('term', items__pid=self.get('pid'))
+            .filter('term', published=True)
+            .sort({'title': {"order": "asc"}})
+            .scan()
```

Comment:
I think we need `.params(preserve_order=True)` here.

Reply:
done